Dockerfile

```diff
@@ -1,31 +1,29 @@
-
 FROM osgeo/gdal:ubuntu-small-3.3.1
 
 ENV DEBIAN_FRONTEND=noninteractive \
     LC_ALL=C.UTF-8 \
-    LANG=C.UTF-8
+    LANG=C.UTF-8 \
+    CIABPASSWORD=secretpassword
```

Comment:
This shouldn't be in the Dockerfile, I don't think.

Reply:
Unless we make it a build argument. Then I agree!

Reply:
Would it not make sense to put it in a .env

Reply:
But then it's fixed in the Dockerfile.

I guess it can be overwritten at run time, but then it doesn't need to be in the Dockerfile up here either. These ENV vars are more for what is needed to build the image, I'd think.